### PR TITLE
[3498] Move notifications link to header

### PR DESCRIPTION
**Pull request by dankmitchell**
Waiting on a method from - https://github.com/DFE-Digital/publish-teacher-training/pull/1178/files

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Product Review

### CODE REVIEW

**Comment by dankmitchell** [comment]
[[S] Move notifications link into header](https://trello.com/c/u5xsq0wf/3498-s-move-notifications-link-into-header)

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -20,6 +20,11 @@
       <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation" data-module="govuk-button">Menu</button>
       <nav>
         <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
+          <% if current_user.present? && current_user['associated_with_accredited_body'] %>
+            <li class="govuk-header__navigation-item">
+              <%= govuk_link_to "Notifications", notifications_path, class: "govuk-header__link", data:{qa: "notifications-link"} %>
+          </li>
```
Comment (aliuk2012) [inline app/views/layouts/_header.html.erb:26]: ident ;)

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -20,6 +20,11 @@
       <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation" data-module="govuk-button">Menu</button>
       <nav>
         <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
+          <% if current_user.present? && current_user['associated_with_accredited_body'] %>
+            <li class="govuk-header__navigation-item">
+              <%= govuk_link_to "Notifications", notifications_path, class: "govuk-header__link", data:{qa: "notifications-link"} %>
```
Comment (aliuk2012) [inline app/views/layouts/_header.html.erb:25]: do we need the `data` attrribute? We are trying to move away from `data-qa` just for testing

Comment (aliuk2012) [reply app/views/layouts/_header.html.erb:25]: We also need an `active` state class on this link if the user is currently on the notifications page

File: app/views/providers/show.html.erb [hunk 3]
```diff
@@ -41,9 +41,6 @@
   </div>
   <aside class="govuk-grid-column-one-third">
     <div class="app-related">
-      <% if @provider_view.show_notifications_link? %>
```
Comment (aliuk2012) [inline app/views/providers/show.html.erb:44]: I'm not sure if we use `@provider_view.show_notifications_link` anywhere else but could we remove https://github.com/DFE-Digital/publish-teacher-training/blob/master/app/view_objects/provider_view.rb#L7-L9 ?

File: spec/site_prism/page_objects/partials/header.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+module PageObjects
+  module Partials
+    class Header < SitePrism::Page
+      element :notifications_preference_link, "[data-qa=notifications-link]"
```
Comment (aliuk2012) [inline spec/site_prism/page_objects/partials/header.rb:4]: We are trying to move away from using `data-qa` attributes for testing. 

What about something like
```suggestion
      element :notifications_preference_link, "a.govuk-header__link", text: "Notifications"
```

Comment (aliuk2012) [reply spec/site_prism/page_objects/partials/header.rb:4]: Could use a similar lookup to test setting the active class styling

**Pull request merged**